package.json

```diff
@@ -35,28 +35,22 @@
     "gulp-util": "3.0.8",
     "gulp-webpack": "^1.5.0",
     "hugo-lunr": "0.0.4",
-    "moment": "^2.20.1",
     "normalize.css": "^7.0.0",
     "normalize.scss": "^0.1.0",
     "path": "0.12.7",
-    "promise-polyfill": "^7.0.2",
     "remove-markdown": "0.2.2",
     "require-dir": "0.3.2",
     "rivet-uits": "^1.3.0",
     "striptags": "3.0.1",
     "tippy.js": "^1.2.0",
-    "vue-loader": "^12.1.0",
-    "vue-template-compiler": "2.4.2",
     "webpack": "^3.10.0"
   },
   "dependencies": {
-    "axios": "0.16.2",
     "clipboard": "^1.7.1",
     "lodash": "4.17.4",
     "lunr": "2.1.2",
     "marked": "0.3.6",
     "remarkable": "^1.7.1",
-    "vee-validate": "^2.0.9",
-    "vue": "2.4.2"
+    "vee-validate": "^2.0.9"
```

Comment:
We can get rid of this `vee-validate` dependency. It's a validation library for Vue that we're no longer using. You can also get rid of:

- `lunr`
- `marked`
- `remarkable`
- `lodash`

As those we all being used in the old Vue search implementation. 

Let's also move `clipboard` to the the `devDependencies` object and get rid of `dependencies` since clipboard is really a development dependency only i.e. it gets compiled down into our final JS bundle. 👍